config/samples/backends/lvm-iscsi/kustomization.yaml

```diff
@@ -4,4 +4,4 @@ resources:
   - lvm.yaml
   - ./../bases
 patches:
-  - backend.yaml
\ No newline at end of file
+  - backend.yaml
```

Comment:
same here what is the diff? Is is spacing/tabs or something like that?

Reply:
i wonder is it removal of the newline? was that intended/needed by linting?

Reply:
this change removed extra line